packages/dbos-cloud/applications/deploy-app-code.ts

```diff
@@ -23,6 +26,11 @@ export async function deployAppCode(host: string, docker: boolean): Promise<numb
   }
   logger.info(`Loaded application name from package.json: ${appName}`)
 
+  if (!isValidApplicationName(appName)) {
```

Comment:
No need to do this in deploy--the registration would have failed server-side before we get here.